assets/js/code-snippets.js

```diff
@@ -0,0 +1,5 @@
+$(function() {
```

Comment:
Very possibly, but for now I'd like to leave as `code-snippets.js`. I'll be adding in the ability to copy the code in an upcoming pull request and would rather make this it's own file specifically for the code snippets portion of the pattern library website.